.gitignore

```diff
@@ -150,8 +150,8 @@ dmypy.json
 .pyre/
 
 poetry.lock
-running_cluster_id.txt
 
-emr/code/*
 tests/data/registry/*
-emr/credentials/
+emr/credentials/ # not needed, can delete this folder
```

Comment:
Is the comment correct?

Reply:
Yes, I have changed the code to copy the credentials from ~/.aws. If the credentials is stored elsewhere, you would specify that in the emr_config under `aws_credentials_location` (in readme)

Reply:
Then delete?